jib-core/src/test/resources/sampleDockerfile

```diff
@@ -4,5 +4,7 @@ COPY libs /app/libs/
 COPY resources /app/resources/
 COPY classes /app/classes/
 
+EXPOSE 1000
+EXPOSE 2000-2010
```

Comment:
Is this a valid instruction? If not, we might want to reconsider supporting this type of syntax in our configuration, and try to keep it as similar to Dockerfile EXPOSE as possible.

Reply:
[Yes](https://stackoverflow.com/a/28721643)

Reply:
Cools

Reply:
What do you think about supporting UDP?

Reply:
That sounds like a useful thing and probably wouldn't be too hard to add.